README.md

```diff
@@ -1,93 +1,417 @@
-This library is under development and is currently in 0.0.x version series.  Breaking changes may be introduced at minor versions in the 0.x range.  Please lock your dependency to a specific version until this library reaches 1.x.
+# Okta Golang management SDK
 
-Need help? [Okta Developer Forum].
+> :warning: Beta alert! This library is in beta. See [release status](#release-status) for more information.
 
-## Configuration
-Configuration details are pulled from 5 different sources. The following is the order the SDK looks for configuration items and builds on top of all the defaults.
+* [Release status](#release-status)
+* [Need help?](#need-help)
+* [Getting started](#getting-started)
+* [Usage guide](#usage-guide)
+* [Configuration reference](#configuration-reference)
+* [Building the SDK](#building-the-sdk)
+* [Contributing](#contributing)
 
-1. Default configuration.
-2. okta.yaml file from ~/.okta directory.
-3. .okta.yaml file from application directory.
-4. Environment variables.
-5. Configuration provided with the Config struct
+This repository contains the Okta management SDK for Golang. This SDK can be used in your server-side code to interact with the Okta management API and
 
-### Creating a New Configuration
-The sdk will provide a default configuration when calling for a new config struct. This will return all okta defaults which you can then build on top of as long as your dont have any other configuration files or options as listed above.
+* Create and update users with the [Users API](https://developer.okta.com/docs/api/resources/users)
+* Add security factors to users with the [Factors API](https://developer.okta.com/docs/api/resources/factors)
+* Manage groups with the [Groups API](https://developer.okta.com/docs/api/resources/groups)
+* Manage applications with the [Apps API](https://developer.okta.com/docs/api/resources/apps)
+* Much more!
 
+We also publish these libraries for Golang:
+
+* [JWT Verifier](https://github.com/okta/okta-jwt-verifier-golang)
+
+You can learn more on the [Okta + Golang](lang-landing) page in our documentation.
+
+## Release status
+
+This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/).
+
+:warning: Beta alert! This library is in beta. Breaking changes may be introduced at minor versions in the 0.x range. Please lock your dependency to a specific version until this library reaches 1.x.
+
+| Version | Status                    |
+| ------- | ------------------------- |
+| 0.x  | :heavy_check_mark: Beta Release               |
+
+The latest release can always be found on the [releases page][github-releases].
+
+## Need help?
+
+If you run into problems using the SDK, you can
+
+* Ask questions on the [Okta Developer Forums][devforum]
+* Post [issues][github-issues] here on GitHub (for code errors)
+
+## Getting started
+
+Follow these steps to install the Okta Golang SDK in your project:
```

Comment:
Will it always be just this one step? In that case, we could simplify to:

---

To install the Okta Golang SDK in your project, run

```
go get github.com/okta/okta-sdk-golang
```